lib/logto_client.dart

```diff
@@ -11,13 +10,30 @@ import '/src/modules/pkce.dart';
 import '/src/modules/token_storage.dart';
 import '/src/utilities/utils.dart' as utils;
 import 'logto_core.dart' as logto_core;
```

Comment:
double check the underscore style is a dart convention

Reply:
yes, we have lint rule applied